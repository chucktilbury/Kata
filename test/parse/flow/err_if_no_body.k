;-------------------------------------------------------------------------------
; no body for if
;-------------------------------------------------------------------------------


func name()() {

    if(comp.name == 0) 
        plart()()
    
    else(1 != 2) {
        goop()()
    }
}