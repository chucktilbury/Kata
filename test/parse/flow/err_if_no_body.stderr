Syntax: /home/chuck/Src/Kata/test/parse/flow/err_if_no_body.k: 9: 14: expected a function body but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_if_no_body.k: 9: 16: expected function body element but got )
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_if_no_body.k: 9: 18: expected function body but got )
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_if_no_body.k: 11: 10: expected module item or end of file but got (
Warning: /home/chuck/Src/Kata/test/parse/flow/err_if_no_body.k: 11: 11: module is empty
